src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery randomBall = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println("Your ball have "
+                    + randomBall.getRandomBall().getColor() + " color and number "
+                    + randomBall.getRandomBall().getNumber());
+        }
```

Comment:
do you understand that here you actually operate with two balls, from the first you get color. from the second number ? :)
just use sout(lottery.getRandomBall())
it should have toString overridden